tests/ote_cli/test_ote_cli_tools_anomaly_classification.py

```diff
@@ -171,9 +171,24 @@ def test_ote_deploy_openvino(template):
     assert run(['python3', '-m', 'pip', 'install', 'wheel'],
                cwd=os.path.join(deployment_dir, 'python'),
                env=collect_env_vars(os.path.join(deployment_dir, 'python'))).returncode == 0
-    assert run(['python3', '-m', 'pip', 'install', 'demo_package-0.0-py3-none-any.whl'],
+    
+    assert check_ote_sdk_commit_hash_in_requirements(os.path.join(deployment_dir, 'python', 'requirements.txt'))
+    
+    # Remove ote_sdk from requirements.txt, since merge commit (that is created on CI) is not pushed to github and that's why cannot be cloned.
+    # Install ote_sdk from local folder instead.
+    # Install the demo_package with --no-deps since, requirements.txt has been embedded to the demo_package during creation.
```

Comment:
So, does that mean, that requirements are somewhat inconsistent in the CI environment?

Reply:
“Embedded” requirements.txt contains consistent commit hash with local cloned state of repo.

but this commit can’t be used to install ote_sdk via “pip install git+https “, because this commit is a merge commit of user’s branch and target branch of PR. This commit is not pushed to OTE.

i added additional test to check that commit in requirements.txt is current commit in local repo